tests/cypress/e2e/issues_prs2/issue_7428_importing_annotation_from_cloud_after_local_import.js

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('Incorrect cloud storage filename used in subsequent import.', () => {
+    const annotationsArchiveNameCloud = 'bazquux';
+    const annotationsArchiveNameLocal = 'foobar';
+    let createdCloudStorageId;
+    const issueId = '7428';
+    const exportFormat = 'CVAT for images';
+    const rectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 400,
+        firstY: 100,
+        secondX: 500,
+        secondY: 200,
+    };
+
+    const serverHost = Cypress.config('baseUrl').includes('3000') ? 'localhost' : 'minio';
+
+    const cloudStorageData = {
+        displayName: 'Demo bucket',
+        resource: 'public',
+        manifest: 'manifest.jsonl',
+        endpointUrl: `http://${serverHost}:9000`,
+    };
+
+    before(() => {
+        createdCloudStorageId = cy.attachS3Bucket(cloudStorageData);
+        cy.goToTaskList();
+        cy.openTaskJob(taskName);
+        cy.createRectangle(rectangleShape2Points);
+        cy.saveJob('PATCH', 200, 'saveJobExportDataset');
+        cy.goToTaskList();
+        cy.openTask(taskName);
+    });
+
+    after(() => {
+        cy.contains('.cvat-header-button', 'Cloud Storages').should('be.visible').click();
+        cy.url().should('include', '/cloudstorages');
+        cy.deleteCloudStorage(cloudStorageData.displayName);
+    });
+
+    describe(`Testing issue "${issueId}"`, () => {
+        it('Export Annotation to the local storage', () => {
+            cy.clickInTaskMenu('Export task dataset', true, taskName);
+            cy.get('.cvat-modal-export-task').should('be.visible').find('.cvat-modal-export-select').click();
+            cy.contains('.cvat-modal-export-option-item', exportFormat).should('be.visible').click();
+            cy.get('.cvat-modal-export-task').find('.cvat-modal-export-select').should('contain.text', exportFormat);
+            cy.get('.cvat-modal-export-task').find('.cvat-modal-export-filename-input').type(annotationsArchiveNameLocal);
+            cy.contains('button', 'OK').click();
+            cy.get('.cvat-notification-notice-export-task-start').should('be.visible');
+            cy.closeNotification('.cvat-notification-notice-export-task-start');
+            cy.waitForDownload();
+        });
+
+        it('Export Annotation to the cloud storage', () => {
+            cy.clickInTaskMenu('Export task dataset', true, taskName);
```

Comment:
There a little bit repeating code in export task datasets. You can use the `exportTask` from [here](https://github.com/opencv/cvat/blob/develop/tests/cypress/support/commands.js#L1169) for exporting task datasets.

It seems the `exportTask` command need modify in export to the cloud storage, if you have the opportunity, let's fix it and be DRY :)